addon/mixins/resolver.js

```diff
@@ -196,6 +193,7 @@ function reopenModule(module) {
 		if (matches && namespaces.includes(matches[1])) {
 			const reopen = this._loadModule(entry);
 
+			/* istanbul ignore else */
```

Comment:
Lower the coverage threshold in `package.json` instead of adding comments. It would be easy to forget that this addon does not accomplish the min requirements if the report always says that is covered 100%.

Reply:
You are right

Reply:
I think it's not a bad idea to explicitly mark a part of the code as not being tested using this kind of comments while keeping the existing config for coverage thresholds. This way the requirements (100% coverage) and the parts not tested are clear.